db/schema.rb

```diff
@@ -264,6 +264,8 @@
     t.datetime "updated_at", null: false
     t.boolean "active", default: true
     t.datetime "deleted_at", precision: nil
+    t.string "official_url"
+    t.jsonb "logo_versions_data"
```

Comment:
@austinjb32 - bro, how this is field used?

Reply:
@mikevic This is the way shrine is setuped inthe project. We need the logo_versions_data field for shrine_resize function to work.
![image](https://github.com/user-attachments/assets/cc38d931-3c21-4053-a2b9-5b7e8583b48a)